modules/install-sno-generate-the-discovery-iso-manually.adoc

```diff
@@ -0,0 +1,150 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+[id="generate-the-discovery-iso-manually_{context}"]
+= Generate the discovery ISO manually
+
+Installing {product-title} on a single node requires a discovery ISO, which you can generate with manually with the following procedure.
+
+.Procedure
+
+. Retrieve the {product-title} client (`oc`):
++
+[source,terminal]
+----
+$ curl -k https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz > oc.tar.gz
+----
++
+[source,terminal]
+----
+$ tar zxf oc.tar.gz
+----
++
+[source,terminal]
+----
+$ chmod +x oc
+----
+
+. Set the {product-title} version:
++
+[source,terminal]
+----
+$ OCP_VERSION=<ocp_version>
```

Comment:
nit, I would at least give an explicit example OCP install version here, save the user going off to copy it. Is `latest-4.9` the actual version? https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp-dev-preview/latest-4.9/ Something like: 
```
<1> {product-title} version, for example, `latest-4.9`
```
This is probably high pedantry.